bfcfg

```diff
@@ -648,22 +655,42 @@ sys_cfg_one_byte()
   fi
 }
 
+sys_cfg_multi_byte()
+{
+  local tmp_file=$1
+  local name=$2
+  local offset=$3
+  local num_bytes=$4
+
+  if [ ${dump_mode} -eq 1 ]; then
+    value=0
+    if [ -e ${tmp_file} ]; then
+      value=$(hexdump -s "${offset}" -n "${num_bytes}" -e '"%d" "\n"' "${tmp_file}")
```

Comment:
Does it take into account the byte order? Have you tested that?

Reply:
Hexdump appears to use the machine's native endianness and formats the decimal output correctly. To test I reset my EFI vars, installed the latest Ubuntu release, and verified the values match the expected defaults:
```
sys: LARGE_ICMC_SIZE=512
sys: CE_THRESHOLD=5000
```
Then I modified the values in UEFI menu and verified they appear as expected:
```
 Large ICMC size            [280]
 CE Threshold               [1300]
...
sys: LARGE_ICMC_SIZE=640
sys: CE_THRESHOLD=4864
```
I can update it so the values are displayed in hex though which is what the UEFI menu input requires.

Reply:
Thanks!